Update/omot_014.txt

```diff
@@ -2151,13 +2151,13 @@ void main()
 //r真相はさておいても、殺された動機が全く一貫していない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "真相はさておいても、殺された動機が全く一貫していない。",
-			NULL, "The motives behind the murders actually weren't consistent at all.", Line_Normal);
+			NULL, "Whatever the truth was, the motives behind the murders actually weren't consistent at all.", Line_Normal);
 	ClearMessage();
 
 //r殺される対象となった理由が、『村にとっての敵』と一致しているだけだ。しかも１年前の主婦は、厳密にはこの枠で扱うのも微妙なほど、村にとってはどうでもいい存在だった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "殺される対象となった理由が、『村にとっての敵』と一致しているだけだ。しかも１年前の主婦は、厳密にはこの枠で扱うのも微妙なほど、村にとってはどうでもいい存在だった。",
-			NULL, "The one thing the murders had in common was the victims being \"an enemy of the village.\" And with enough effort, you could reframe the housewife murder a year ago to seem like it benefited the village.", Line_Normal);
+			NULL, "The one thing the murders had in common was the victims being \"an enemy of the village.\" And strictly speaking, it was doubtful if this logic applies to the housewife's case because she didn't really matter to the village.", Line_Normal);
```

Comment:
```suggestion
			NULL, "The one thing the murders had in common was the victims being \"an enemy of the village.\" And strictly speaking, it was doubtful if that logic even applied in the housewife's case, since she didn't really matter to the village.", Line_Normal);
```